client/src/components/MediaCard/MediaCard.stories.js

```diff
@@ -1,34 +1,34 @@
-  .add("Van", () => <MediaCard size='medium' to='User/:userid/' imageSrc={van} footerText='Posted By: --user--' />)
-  .add("Night Sky", () => <MediaCard size='medium' to='User/:userid/' imageSrc={nightSky} footerText='Posted By: --user--' />)
-  .add("Waterfall", () => <MediaCard size='medium' to='User/:userid/' imageSrc={waterfall} footerText='Posted By: --user--' />)
-  .add("Flight", () => <MediaCard size='medium' to='User/:userid/' imageSrc={flight} footerText='Posted By: --user--' />)
+// storiesOf('Medium Media Cards', module)
+//   .add("Default", () => <MediaCard />)
+//   .add("Books", () => <MediaCard size='medium' to='User/:userid/' imageSrc={books} footerText='Posted By: --user--' />)
+//   .add("Happiness", () => <MediaCard size='medium' to='User/:userid/' imageSrc={happiness} footerText='Posted By: --user--' />)
+//   .add("Van", () => <MediaCard size='medium' to='User/:userid/' imageSrc={van} footerText='Posted By: --user--' />)
+//   .add("Night Sky", () => <MediaCard size='medium' to='User/:userid/' imageSrc={nightSky} footerText='Posted By: --user--' />)
+//   .add("Waterfall", () => <MediaCard size='medium' to='User/:userid/' imageSrc={waterfall} footerText='Posted By: --user--' />)
+//   .add("Flight", () => <MediaCard size='medium' to='User/:userid/' imageSrc={flight} footerText='Posted By: --user--' />)
   
-storiesOf('Large Media Cards', module)
-  .add("Books", () => <MediaCard size='large' to='User/:userid/' imageSrc={books} footerText='Posted By: --user--' />)
-  .add("Happiness", () => <MediaCard size='large' to='User/:userid/' imageSrc={happiness} footerText='Posted By: --user--' />)
-  .add("Van", () => <MediaCard size='large' to='User/:userid/' imageSrc={van} footerText='Posted By: --user--' />)
-  .add("Night Sky", () => <MediaCard size='large' to='User/:userid/' imageSrc={nightSky} footerText='Posted By: --user--' />)
-  .add("Waterfall", () => <MediaCard size='large' to='User/:userid/' imageSrc={waterfall} footerText='Posted By: --user--' />)
-  .add("Flight", () => <MediaCard size='large' to='User/:userid/' imageSrc={flight} footerText='Posted By: --user--' />)
\ No newline at end of file
+// storiesOf('Large Media Cards', module)
+//   .add("Books", () => <MediaCard size='large' to='User/:userid/' imageSrc={books} footerText='Posted By: --user--' />)
+//   .add("Happiness", () => <MediaCard size='large' to='User/:userid/' imageSrc={happiness} footerText='Posted By: --user--' />)
+//   .add("Van", () => <MediaCard size='large' to='User/:userid/' imageSrc={van} footerText='Posted By: --user--' />)
+//   .add("Night Sky", () => <MediaCard size='large' to='User/:userid/' imageSrc={nightSky} footerText='Posted By: --user--' />)
+//   .add("Waterfall", () => <MediaCard size='large' to='User/:userid/' imageSrc={waterfall} footerText='Posted By: --user--' />)
+//   .add("Flight", () => <MediaCard size='large' to='User/:userid/' imageSrc={flight} footerText='Posted By: --user--' />)
```

Comment:
What happened here?

Reply:
For whatever reason, this was throwing an error for me. Storybook wouldn't work for me because it couldn't find the assets. I did a pull from develop but it didn't fix it. I forgot to undo my comments! Sorry.

Reply:
Of course, now that I commented it back it its suddenly finding those assets. Cool. At least that means there's nothing wrong!

Reply:
@shelbyvjacobs were you able to fix it? If so, can you push?